jupyterlite_xeus/add_on.py

```diff
@@ -86,6 +88,13 @@ class XeusAddon(FederatedExtensionAddon):
         description="The path to the wasm prefix",
     )
 
+    mount_jupyterlite_content = Bool(
+        None,
+        allow_none=True,
+        config=True,
+        description="Whether or not to mount the jupyterlite content into the kernel. This allows bypassing the file-system core JupyterLite implementation.",
```

Comment:
Does it completely bypass, or does it provide a way to bundle additional files at build time?

Reply:
Answering my own question after checking the code below: it does seem to bypass:

```ts
if (await this._remote.isDir('/files')) {
    await this._remote.cd('/files');
} else if (options.mountDrive) {
    await this._remote.cd(localPath);
}
```

Reply:
> Does it completely bypass, or does it provide a way to bundle additional files at build time?

You can still provide extra mount points 👍🏽 It's just that now jupyterlite-xeus has an extra mount point `/files` that contains the jupyterlite content.

And yes the service worker is still enabled. So if you do an `os.chdir` into the mount point of the service worker based filesystem, you'd be using that.

Reply:
Indeed it would be nice to clarify all this in proper documentation

Reply:
ah ok, thanks for the extra context :+1: